docs/design/coordinator.md

```diff
@@ -73,6 +73,13 @@ that are removed from the database.
 Segments that are overshadowed (their versions are too old and their data has been replaced by newer segments) are
 marked as unused. During the next Coordinator's run, they will be unloaded from Historical nodes in the cluster.
 
+### Cleaning up dangling segments
+
+On each run, the Druid coordinator will determine and clean up dangling segments for each datasource. Dangling segments must
+satisfy the following criteria:
+- It is a tombstone segment that starts at -INF or ends at INF (e.g., a tombstone with an interval of `-146136543-09-08T08:23:32.096Z/2000-01-01` or `2020-01-01/146140482-04-24T15:36:27.903Z`) and
+- It does not overlap with any used segment
```

Comment:
Minor copyedit suggestion since it's a little odd to say satisfy since that implies it is an active process vs something like fit or match
```suggestion
### Clean up dangling segments

On each run, the Druid coordinator determines and cleans up dangling segments for each datasource. Dangling segments are segments that fit the following criteria:
- It is a tombstone segment that starts at -INF or ends at INF (e.g., a tombstone with an interval of `-146136543-09-08T08:23:32.096Z/2000-01-01` or `2020-01-01/146140482-04-24T15:36:27.903Z`) and
- It does not overlap with any used segment
```